core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -342,9 +342,17 @@ export class Topic {
   }
 
   // Adds a new frontend subtopic domain object to this topic.
-  addSubtopic(title: string): void {
-    let newSubtopic = Subtopic.createFromTitle(
-      this._nextSubtopicId, title);
+  addSubtopic(_title: string): void {
+    let subtopicBackendDict: SubtopicBackendDict = {
+      id: this._nextSubtopicId,
```

Comment:
Don't do this -- you should keep this method unchanged from before. Only create the new subtopic object for the first time when all fields are filled and the green button is clicked. Until then, keep the fields in the modal as individual fields.